src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorSingleStream.java

```diff
@@ -65,7 +64,7 @@
 
     private long lastProcessTime = -1L;
 
-    private final RemoteBitrateObserver observer;
+    private final VideoMediaStreamImpl stream;
```

Comment:
Will do as you suggest. Just a remark, the observer doesn't do anything actually, i.e. we're not acting on any observations, so we may want to completely remove this in the future.